docs/dev-guide/plugin-dev.md

```diff
@@ -234,18 +234,19 @@ api.onCreateComplete(() => {
 
 Finally, you need to write the content back to the main file:
 
-```js{11}
+```js{2,11}
 // generator/index.js
 
 api.onCreateComplete(() => {
+  const { EOL } = require('os')
   const fs = require('fs')
   const contentMain = fs.readFileSync(api.entryFile, { encoding: 'utf-8' })
   const lines = contentMain.split(/\r?\n/g)
 
   const renderIndex = lines.findIndex(line => line.match(/render/))
   lines[renderIndex] += `\n  router,`
```

Comment:
Now that we join the lines with `os.EOL`, this string concatenation has to be changed too:
```suggestion
   lines[renderIndex] += `${EOL}  router,`
```
or
```suggestion
   lines.splice(renderIndex, 0, `  router,`)
```